docs/site/LICENSE

```diff
@@ -0,0 +1,12 @@
+Attribution 3.0 Unported (CC BY 3.0)
```

Comment:
Hey @carolalynn22 - the general guidance from the ASWF TAC is to leverage the CC-BY-40 license for non-code contributions ( http://creativecommons.org/licenses/by/4.0/ ). Not sure if there is a specific driver behind this choice or not - happy to discuss further if you'd like.

Reply:
I believe this license is from the theme.

Reply:
Ah ok - in that case we should leave this as-is.